AutomatedTesting/Gem/PythonTests/assetpipeline/scene_tests/scene_tests.py

```diff
@@ -0,0 +1,185 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+import os
+import pytest
+import logging
+import re
+from pprint import pformat
+from ly_test_tools.o3de.asset_processor import ASSET_PROCESSOR_PLATFORM_MAP
+
+from assetpipeline.ap_fixtures.asset_processor_fixture import asset_processor as asset_processor
+from assetpipeline.ap_fixtures.ap_setup_fixture import ap_setup_fixture as ap_setup_fixture
+
+from automatedtesting_shared import asset_database_utils as asset_db_utils
+import assetpipeline.ap_fixtures.test_builder as test_builder
+import assetpipeline.ap_fixtures.test_debug_parse_and_compare as parse_and_compare
+import ly_test_tools.o3de.pipeline_utils as utils
+
+logger = logging.getLogger(__name__)
+targetProjects = ["AutomatedTesting"]
+
+
+# Helper: Gets a case correct version of the cache folder
+def get_cache_folder(asset_processor):
+    # Make sure the folder being checked is fully lowercase.
+    # Leave the "c" in Cache uppercase.
+    return re.sub("ache[/\\\\](.*)", lambda m: m.group().lower(), asset_processor.project_test_cache_folder())
+
+
+@pytest.fixture
+def local_resources(request, workspace, ap_setup_fixture):
+    ap_setup_fixture["tests_dir"] = os.path.dirname(os.path.realpath(__file__))
+
+
+# Set the path of the directory containing the tests, then parametrize the tests using the test_builder
+dir_test_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), "tests")
+blackbox_scene_tests, blackbox_test_ids = test_builder.parametrize_blackbox_scene_test(dir_test_path)
+
+
+@pytest.mark.usefixtures("asset_processor")
+@pytest.mark.usefixtures("ap_setup_fixture")
+@pytest.mark.usefixtures("local_resources")
+@pytest.mark.parametrize("project", targetProjects)
+@pytest.mark.assetpipeline
+@pytest.mark.SUITE_sandbox
+class TestScene_AllPlatforms(object):
+
+    @pytest.mark.BAT
+    @pytest.mark.parametrize("blackbox_param", blackbox_scene_tests, ids=blackbox_test_ids)
+    def test_SceneBlackboxTest_SourceFiles_Processed_ResultInExpectedProducts(self, workspace, ap_setup_fixture,
+                                                                              asset_processor, project, blackbox_param):
+        """
+        Please see run_fbx_test(...) for details
+        Test Steps:
+        1. Determine if blackbox is set to none
+        2. Run FBX Test
+        3. Determine if override assets is set to none
+        4. If not, re-run FBX test and validate the information in override assets
+        """
+
+        if blackbox_param is None:
```

Comment:
should this error out or something? Seems strange to have this test get invoked, do nothing when blackbox_param is None, and pass.